src/api/app/views/webui2/webui/user/index.html.haml

```diff
@@ -4,42 +4,23 @@
   = render partial: 'webui/configuration/tabs'
   .card-body
     %h3= @pagetitle
-    %table.responsive.table.table-sm.table-striped.table-bordered#user-table
+    %table.responsive.table.table-sm.table-striped.table-bordered#user-table{ data: { source: users_path } }
       %thead
         %tr
-          %td
+          %th
             User
-          - if Configuration.ldap_enabled?
```

Comment:
Should this `if` be removed? It seems like it should not.

Reply:
It should. The visibility of this column is now set during the [datatable initialization](https://github.com/openSUSE/open-build-service/pull/7108/files#diff-c76d56f625bcb8033c7090e2efe29d64R8)

Reply:
Oh okay, nice!